### PR TITLE
Defaulted one_click_purchase_enabled to True

### PR DESCRIPTION
**Pull request by staubina**
[LEARNER-4061]

A small change to make `one_click_purchase_enabled` default to True.

### CODE REVIEW

**Review by McKenzieW — APPROVED** [review]

File: course_discovery/apps/course_metadata/migrations/0079_auto_20180319_1718.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (Mjloturco) [inline course_discovery/apps/course_metadata/migrations/0079_auto_20180319_1718.py]: You could give this migration a meaningful name

**Review by Mjloturco — APPROVED** [review]
could add a test that verifies on new program creation that 1-click is enabled, or just an assertion to an existing creation test. otherwise LGTM

**Comment by staubina** [comment]
> could add a test that verifies on new program creation that 1-click is enabled

I am not sure I see the value in adding a test as it is part of the model.  

> You could give this migration a meaningful name

I will take a look at renaming the migration.

**Comment by staubina** [comment]
@Mjloturco I renamed the migration per your request.

**Pull request merged**